integration-cli/docker_cli_by_digest_test.go

```diff
@@ -441,9 +344,7 @@ func (s *DockerRegistrySuite) TestPsListContainersFilterAncestorImageByDigest(c
 
 	// Invalid imageReference
 	out, _ = dockerCmd(c, "ps", "-a", "-q", "--no-trunc", fmt.Sprintf("--filter=ancestor=busybox@%s", digest))
-	if strings.TrimSpace(out) != "" {
-		c.Fatalf("Expected filter container for %s ancestor filter to be empty, got %v", fmt.Sprintf("busybox@%s", digest), strings.TrimSpace(out))
-	}
+	c.Assert(strings.TrimSpace(out), checker.Equals, "", check.Commentf("Expected filter container for %s ancestor filter to be empty, got %v", fmt.Sprintf("busybox@%s", digest), strings.TrimSpace(out)))
```

Comment:
No need for the extras in the commentf, can just move the actual comment text in a normal go comment above.